README.md

```diff
@@ -123,3 +123,13 @@ nmake
 Note: Using the NMake Makefiles generator is important, since the normal
 Visual Studio generator does not work well in combination with some of
 the build systems of the sub-projects that are built.
+
+
+### Docker
+Building the environment in Docker is very easy using the Dockerfile in this repository.
+We handle all cloning and compiling in the Dockerfile, so no dependencies are needed on the host OS.
+The resulting image can be used as a `FROM` for building Cura itself.
+
+```bash
+docker build -t ultimaker/cura-build-environment:1 .
```

Comment:
I think the tag `1` is not necessary here.

Reply:
It's a specific version of the image (now 1).

Reply:
Fixed in latest commit.